posts/faq/how-to-report-bugs.md

```diff
@@ -0,0 +1,5 @@
+---
+title: How do I report bugs?
+---
+
+Have a technical problem? If you're having an issue with install of the UI/miner, or something involving an invite code, please join the slack at https://lbry.slack.com and in channel #bugs post a SINGLE detailed note with what exactly is going on.  If you can get to the LBRY UI after installing, but can't watch a video or have another problem, please click the three horizontal lines in the top right hand corner of the LBRY UI, click Help, then click the Send Us a Bug Report link.
```

Comment:
Please include this too:

"If you need an invite to the LBRY slack, you can get one at https://slack.lbry.io".